### PR TITLE
OKTA-658529 dev org minus org2org

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Dev orgs not supporting Org2Org app integration
- **Is this PR related to a Monolith release?** 2023.12.1 (Dec 19, 2023)

### Preview:
https://65809db358f3e60c4895b78d--reverent-murdock-829d24.netlify.app/docs/guides/secure-oauth-between-orgs/main/

### Resolves:

* [OKTA-658529](https://oktainc.atlassian.net/browse/OKTA-658529)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/concepts/multi-tenancy/index.md](https://github.com/okta/okta-developer-docs/blob/45880d057aef12a3a20f050b5a660d892bbc575a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fmulti-tenancy%2Findex.md) | 81 | 58 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b9fa8312-0170-47b7-a0df-e429a70b4b5c) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md](https://github.com/okta/okta-developer-docs/blob/45880d057aef12a3a20f050b5a660d892bbc575a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsecure-oauth-between-orgs%2Fmain%2Findex.md) | 80 | 68 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/fcccf963-bf94-40b1-b101-8dbeec4a8348) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/api/apps/index.md](https://github.com/okta/okta-developer-docs/blob/45880d057aef12a3a20f050b5a660d892bbc575a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fapps%2Findex.md) | 69 | 641 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8b3a1caf-b0ab-442e-9387-936b668fcd23) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 3 of 3 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.c18dfef9-f2bb-4573-9d0c-049f324ee886?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/concepts/multi-tenancy/index.md](https://github.com/okta/okta-developer-docs/blob/96cd238b45c3eff5cf747b70bb17062e9b4af409/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fmulti-tenancy%2Findex.md) | 81 | 58 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/443400d7-8ff7-452f-bab9-69c12141d6b8) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md](https://github.com/okta/okta-developer-docs/blob/96cd238b45c3eff5cf747b70bb17062e9b4af409/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsecure-oauth-between-orgs%2Fmain%2Findex.md) | 80 | 68 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/cf9537f4-1d63-4703-97ad-06a70adb3950) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/api/apps/index.md](https://github.com/okta/okta-developer-docs/blob/96cd238b45c3eff5cf747b70bb17062e9b4af409/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fapps%2Findex.md) | 69 | 641 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c6935ace-3dc1-4fa7-a16f-9012c6e4c869) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 3 of 3 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.4bd2ef19-1f1a-4564-b90f-8805c65e42ca?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/concepts/multi-tenancy/index.md](https://github.com/okta/okta-developer-docs/blob/ecf73e7993a3e33b789ebaa8cd248d87879b0421/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fmulti-tenancy%2Findex.md) | 81 | 59 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/058c31f4-fee7-4e77-9741-86ae6b53a310) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md](https://github.com/okta/okta-developer-docs/blob/ecf73e7993a3e33b789ebaa8cd248d87879b0421/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsecure-oauth-between-orgs%2Fmain%2Findex.md) | 80 | 70 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b7a9458a-12ef-458b-a4d2-24178a5e3d8a) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/api/apps/index.md](https://github.com/okta/okta-developer-docs/blob/ecf73e7993a3e33b789ebaa8cd248d87879b0421/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fapps%2Findex.md) | 69 | 642 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/074bbd16-1767-4cba-a91a-ee6cdba23f0f) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 3 of 3 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.7f7d070d-ffdc-481f-8d0b-24e8d1e59b7a?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Review by jefftaylor-okta — APPROVED** [review]
Approved!

File: packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md [hunk 1]
```diff
@@ -4,55 +4,71 @@ excerpt: Learn how to synchronize users and groups between orgs with OAuth 2.0 i
...
-This guide explains how to securely set up Okta hub and spoke orgs to synchronize users and groups by using OAuth 2.0 in a [multi-tenant solution](/docs/concepts/multi-tenancy/). The connection between Okta orgs can be set up for API access to specific scoped data using the OAuth 2.0 client credential flow for the [Org2Org integration](https://help.okta.com/okta_help.htm?type=oie&id=ext-org2org-intg) app.
+This guide explains how to securely configure Okta hub-and-spoke orgs to synchronize users and groups by using OAuth 2.0 in a [multi-tenant solution](/docs/concepts/multi-tenancy/). The connection between Okta orgs is configured as an OAuth 2.0 [Org2Org app integration](https://help.okta.com/okta_help.htm?type=oie&id=ext-org2org-intg). This connection provides API access to scoped data using the OAuth 2.0 Client Credential flow.
 
 > **Note**: Currently, only the Okta API can be used to enable OAuth 2.0-based provisioning.
 ---
 
 **Learning outcomes**
 
-* Set up a multi-tenant hub and spoke Okta org provisioning configuration to use the OAuth 2.0 client credential grant flow.
+* Set up a hub-and-spoke Okta org provisioning to use the OAuth 2.0 Client Credential grant flow.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md:14]:
```suggestion
* Set up a hub-and-spoke Okta org provisioning to use the OAuth 2.0 Client Credentials grant flow.
```

File: packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md [hunk 2]
```diff
@@ -4,55 +4,71 @@ excerpt: Learn how to synchronize users and groups between orgs with OAuth 2.0 i
...
+
+To secure API connections between orgs for a [multi-tenant solution](/docs/concepts/multi-tenancy/) that uses the hub-and-spoke model, use the Okta Org2Org integration as an OAuth 2.0 client.
 
-To secure API connections between orgs for a [multi-tenant solution](/docs/concepts/multi-tenancy/) that uses the hub and spoke model, Okta provides the OAuth 2.0 client credential flow using the Okta Org2Org integration as an OAuth 2.0 client. This enables Okta admins to push and match both users and groups from one Okta org to another. Previously, this integration only supported token-based access to the Okta API. Now the Org2Org integration can be configured to access the Okta API as an OAuth 2.0 client. This increases security by limiting the scope of access and providing a better mechanism to rotate credentials.
+Previously, the Org2Org integration only supported token-based access to the Okta API.
+The Org2Org integration can now be configured to access Okta APIs as an OAuth 2.0 client using the OAuth 2.0 Client Credential flow. The OAuth 2.0 API access enables you to increase security by limiting the scope of resource access and providing a better mechanism to rotate credentials.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md:33]:
```suggestion
You can now configure the Org2Org integration to access Okta APIs as an OAuth 2.0 client using the OAuth 2.0 Client Credentials flow. The OAuth 2.0 API access enables you to increase security by limiting the scope of resource access and providing a better mechanism to rotate credentials.
```

File: packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md [hunk 3]
```diff
@@ -4,55 +4,71 @@ excerpt: Learn how to synchronize users and groups between orgs with OAuth 2.0 i
...
+In this configuration:
+
+* The hub org acts as the resource server and the [authorization server](/docs/concepts/auth-servers/)
+
+* Each spoke org represents a [service app in the hub org for the OAuth 2.0 Client Credentials flow](/docs/guides/implement-oauth-for-okta-serviceapp/main/)
+
+* At the spoke org level, an Org2Org app represents the OAuth 2.0 client
 
-After the Org2Org OAuth 2.0 connection is configured, you can start to test your connection and push user data from the spoke orgs to the hub org. You can also create a [rotate keys](#key-rotation) schedule to update your credentials on a regular basis.
+* To limit access to the OAuth 2.0 clients, the hub-org service app needs to be granted with allowed [OAuth 2.0 scopes](https://developer.okta.com/docs/api/oauth2/#okta-admin-management)
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md:49]:
```suggestion
* To limit access to the OAuth 2.0 clients, the hub-org service app needs to be granted with allowed [OAuth 2.0 scopes](https://developer.okta.com/docs/api/oauth2/#okta-admin-management).
```

File: packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md [hunk 4]
```diff
@@ -4,55 +4,71 @@ excerpt: Learn how to synchronize users and groups between orgs with OAuth 2.0 i
...
 
-> **Note**: For provisioning in a hub and spoke org model that uses API tokens instead of OAuth 2.0, see [Integrate Okta Org2Org with Okta ](https://help.okta.com/okta_help.htm?type=oie&id=ext-org2org-intg).
+After the OAuth 2.0 connection is configured, start to test your connection: push user data from the spoke orgs to the hub org. You can also create a [Rotate key](#key-rotation) schedule to update your credentials regularly.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md:51]:
```suggestion
After you configure the OAuth 2.0 connection, test your connection: push user data from the spoke orgs to the hub org. You can also create a [Rotate key](#key-rotation) schedule to update your credentials regularly.
```

File: packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md [hunk 5]
```diff
@@ -389,11 +407,11 @@ curl -X PUT \
...
+> **Note**: Specify all the required parameters when you update a client app. Partial updates aren't supported. If any mandatory parameters are missing when you update a client app, the update fails. When you update the keys in the service app `jwks` parameter, all the old keys are overwritten. To add a key and keep the old key, you need to specify both old and new keys in the `jwks.keys` array.
 
 ### Update the current credentials for the Org2Org app
 
-From the response of the [previous generate key POST request](#generate-a-new-key-for-the-org2org-app), copy the `kid` property and [activate the new key by updating the Org2Org app](/docs/reference/api/apps/#update-key-credential-for-application).
+From the response of the [previous generate key POST request](#generate-a-new-key-for-the-org2org-app), copy the `kid` property, and [activate the new key by updating the Org2Org app](/docs/reference/api/apps/#update-key-credential-for-application).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md:414]:
```suggestion
From the response of the [previous generate key POST request](#generate-a-new-key-for-the-org2org-app), copy the `kid` property and [activate the new key by updating the Org2Org app](/docs/reference/api/apps/#update-key-credential-for-application).
```

**Review by susanharper-okta — APPROVED** [review]
couple nits - approving to not hold you up.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/concepts/multi-tenancy/index.md](https://github.com/okta/okta-developer-docs/blob/58e85412ffe2b94264b6ba4c3de6034a3a0bddee/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fmulti-tenancy%2Findex.md) | 81 | 59 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/902be775-25a1-47d5-ae13-b89f965b5c17) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md](https://github.com/okta/okta-developer-docs/blob/58e85412ffe2b94264b6ba4c3de6034a3a0bddee/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsecure-oauth-between-orgs%2Fmain%2Findex.md) | 80 | 70 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/43e25bd7-e97f-46da-a3ad-95beac684b34) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/api/apps/index.md](https://github.com/okta/okta-developer-docs/blob/58e85412ffe2b94264b6ba4c3de6034a3a0bddee/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fapps%2Findex.md) | 69 | 642 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/3fb0d93d-f54d-4c23-998f-b82d9f98e628) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 3 of 3 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.3c7d85c6-909a-4534-ac27-baa0e6e7c4c4?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/concepts/multi-tenancy/index.md](https://github.com/okta/okta-developer-docs/blob/94e23ea70e55003b9e2492b1ed914996122d6355/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fmulti-tenancy%2Findex.md) | 81 | 59 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/fc8fc06c-034f-42d1-832f-094e3cdb7db8) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md](https://github.com/okta/okta-developer-docs/blob/94e23ea70e55003b9e2492b1ed914996122d6355/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsecure-oauth-between-orgs%2Fmain%2Findex.md) | 80 | 71 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/94b50c48-8fab-4eda-afc2-7f4cdc88bfd8) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/api/apps/index.md](https://github.com/okta/okta-developer-docs/blob/94e23ea70e55003b9e2492b1ed914996122d6355/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fapps%2Findex.md) | 69 | 642 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/86758c33-2e45-43b9-b9a4-790b4d710876) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 3 of 3 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.99a82d3b-eb3e-4668-beea-c92d8d4342b6?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/concepts/multi-tenancy/index.md](https://github.com/okta/okta-developer-docs/blob/4c391b7fb34e79734a01f4e1037897577619f85a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fmulti-tenancy%2Findex.md) | 81 | 59 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/970120e8-098f-457e-ab5d-56ed4a111bda) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md](https://github.com/okta/okta-developer-docs/blob/4c391b7fb34e79734a01f4e1037897577619f85a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsecure-oauth-between-orgs%2Fmain%2Findex.md) | 80 | 71 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/bbdb9bd3-8023-4504-acd2-ed9ec23a08e3) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/api/apps/index.md](https://github.com/okta/okta-developer-docs/blob/4c391b7fb34e79734a01f4e1037897577619f85a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fapps%2Findex.md) | 69 | 642 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/7ea6f016-df8b-45ee-9a93-9d542f2e9662) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 3 of 3 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.6db68410-b8f3-443f-9c91-2f3c99486817?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/concepts/multi-tenancy/index.md](https://github.com/okta/okta-developer-docs/blob/b6c689bd66adbe66fa0c354bfe305a33bc9e32ed/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fmulti-tenancy%2Findex.md) | 81 | 59 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/ca777a79-a1a5-4de3-b1de-858f5f163f86) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md](https://github.com/okta/okta-developer-docs/blob/b6c689bd66adbe66fa0c354bfe305a33bc9e32ed/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsecure-oauth-between-orgs%2Fmain%2Findex.md) | 79 | 72 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/9db47b62-2a09-4e45-a7cb-e641998d3093) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/api/apps/index.md](https://github.com/okta/okta-developer-docs/blob/b6c689bd66adbe66fa0c354bfe305a33bc9e32ed/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fapps%2Findex.md) | 69 | 642 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/438e6775-b1f3-451c-9e6d-288ae131738a) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 3 of 3 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.83c9a1ae-0166-4a94-be22-2779b4fb51b3?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/concepts/multi-tenancy/index.md](https://github.com/okta/okta-developer-docs/blob/492bccfcc6f1ffc211279a2d3f9d1af48068ad70/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fmulti-tenancy%2Findex.md) | 81 | 59 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/4c7165a8-b6b8-4c25-a16d-71a4811e4e39) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md](https://github.com/okta/okta-developer-docs/blob/492bccfcc6f1ffc211279a2d3f9d1af48068ad70/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsecure-oauth-between-orgs%2Fmain%2Findex.md) | 79 | 72 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/50d06581-4958-41f2-b47b-794f97be4adc) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/api/apps/index.md](https://github.com/okta/okta-developer-docs/blob/492bccfcc6f1ffc211279a2d3f9d1af48068ad70/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fapps%2Findex.md) | 69 | 642 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/85332cb8-41fb-4f7f-b973-c56f3a9ccf55) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 3 of 3 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.3147b27f-3020-4449-a4c9-8f1f73727de7?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Pull request merged**